web/test.sh

```diff
@@ -124,11 +125,13 @@ if builder_start_action test:engine; then
     KARMA_FLAGS="$KARMA_FLAGS --reporters $REPORTERS"
   fi
 
+  KARMA_EXT_FLAGS=
   if ! builder_has_option --ci; then
-    KARMA_FLAGS="$KARMA_FLAGS --browsers $BROWSERS"
+    KARMA_EXT_FLAGS="$KARMA_FLAGS --browsers $BROWSERS"
   fi
 
-  npm --no-color run karma -- start $KARMA_FLAGS src/test/auto/integrated/$CONFIG
+  npm --no-color run karma -- start $KARMA_FLAGS src/test/auto/dom/$CONFIG
+  npm --no-color run karma -- start $KARMA_FLAGS $KARMA_EXT_FLAGS src/test/auto/integrated/$CONFIG
```

Comment:
```suggestion
  karma start $KARMA_FLAGS src/test/auto/dom/$CONFIG
  karma start $KARMA_FLAGS $KARMA_EXT_FLAGS src/test/auto/integrated/$CONFIG
```

Possibly? Although wondering about the `--no-color` flag here

Reply:
I was wondering about that as well.  I certainly don't remember why it was added.

Reply:
I guess we try taking it out and seeing what terrible things happen as a result :grin: